apps/framework-cli/src/cli/watcher.rs

```diff
@@ -387,79 +387,82 @@ async fn watch(
                             format!("Processing error occurred: {}", e),
                         )
                     })?;
-                }
+                } else {
+                    let framework_object_versions = framework_object_versions.as_mut().unwrap();
```

Comment:
why is unwrap safe here?

Reply:
Because framework_object_versions is passed as Some when not core_v2.

This is temp and that code will be removed once we do the move ove